joint_state_publisher/joint_state_publisher.py

```diff
@@ -136,33 +135,34 @@ def __init__(self, node, description):
         self.node = node
         self.free_joints = {}
         self.joint_list = [] # for maintaining the original order of the joints
-        self.dependent_joints = self.get_param('dependent_joints', {})
-        self.use_mimic = self.get_param('use_mimic_tags', True)
-        self.use_small = self.get_param('use_smallest_joint_limits', True)
 
-        self.zeros = self.get_param('zeros')
+        self.dependent_joints = {}
```

Comment:
Is it still the case in Dashing that we can't get maps from the Python layer?  I thought we had made Python parameters mostly feature-parity with C++, but I'll admit I didn't follow it too closely.  If it is the case that we can't do map parameters in Python, then I'm fine with disabling `dependent_joints` and `zeros` for now, but I'd like a `TODO` comment to re-enable once it is available.

Reply:
That does not appear to be the case:
https://github.com/ros2/rclpy/blob/a28b8258c2a1f6300bb407b2d7e10e4e98b60b48/rclpy/rclpy/parameter.py#L25-L35

I'll add a TODO.

Reply:
> That does not appear to be the case:
> https://github.com/ros2/rclpy/blob/a28b8258c2a1f6300bb407b2d7e10e4e98b60b48/rclpy/rclpy/parameter.py#L25-L35

To be clear, in C++ we don't have a dedicated type for maps either.  Instead, we have individual parameters of the form `prefix.key = value`, and then there are some methods to get everything that starts with `prefix` in a map.  It's possible we have a similar thing in Python, but I just don't know; @ivanpauno, could you chime in here and let us know if parameter maps are supported in Python?  If so, are there examples we can look at?

Reply:
In rclcpp, we have this method: https://github.com/ros2/rclcpp/blob/e7c463dc74849aa64cb8a74e13f2a57b4ed59862/rclcpp/include/rclcpp/node.hpp#L783-L787
I haven't found an equivalent in rclpy.

Opened an issue: https://github.com/ros2/rclpy/issues/374

Note: In the rclcpp method, all the parameters with the same prefix should be of the same type. In python this could be avoided, but I would restrict it in the same way, in order to have the same behavior.